crates/solvers/src/infra/dex/balancer/dto.rs

```diff
@@ -0,0 +1,155 @@
+use {
```

Comment:
This is just copied from the `balancer_sor_api.rs` file. One notable exception 👇.